### ISSUE TITLE
v3.0 compatibility

### ISSUE THREAD

**Issue by yzchai** [eliasdorneles/code-igniter-cas-library#3]
Hi,

does anyone have any problem with this library using the Code Ignitor v3.x? seems like the session is not saved when the redirect happen. So it goes on an inifinite loops

**Comment by rmoix** [comment]
Yes, I do have the same problem. It worked with CI2.x, but not with 3.0.4. Once the session is loaded in the autoload configuration file I get the following error:

```
A PHP Error was encountered

Severity: Error

Message: session_start() [function.session-start]: Failed to initialize storage module: user (path: C:\Windows\Temp)

Filename: CAS/Client.php

Line Number: 3588

Backtrace:
```

**Comment by eliasdorneles** [comment]
Hey fellows!
I haven't been working neither with PHP, CodeIgniter or CAS for a while, so I don't have an immediate way of verifying what's going on.
I skimmed through the the upgrade notes for CI 3, have you followed the instructions there for configuring the session usage?
https://www.codeigniter.com/userguide3/installation/upgrade_300.html#step-6-update-your-session-library-usage
